package.json

```diff
@@ -105,6 +107,7 @@
     "connector": "/",
     "connectSystemsModal": "/connect-systems",
     "logModal": "/log",
-    "activationKeys": "/activation-keys"
+    "activationKeys": "/activation-keys",
```

Comment:
Yeah, it was this way before, moving it will require refactoring some components not related to this story. I'll consider doing it as a separate story